SpatialGDK/Source/SpatialGDK/Private/Interop/WellKnownEntitySystem.cpp

```diff
@@ -8,34 +8,37 @@ DEFINE_LOG_CATEGORY(LogWellKnownEntitySystem);
 
 namespace SpatialGDK
 {
-WellKnownEntitySystem::WellKnownEntitySystem(const FSubView& SubView, USpatialReceiver* InReceiver, USpatialWorkerConnection* InConnection,
+WellKnownEntitySystem::WellKnownEntitySystem(const FSubView& InGdkSubView, const FSubView& InSystemEntitySubView,
+											 USpatialReceiver* InReceiver, USpatialWorkerConnection* InConnection,
 											 const int InNumberOfWorkers, SpatialVirtualWorkerTranslator& InVirtualWorkerTranslator,
 											 UGlobalStateManager& InGlobalStateManager)
-	: SubView(&SubView)
+	: GdkSubView(&InGdkSubView)
```

Comment:
Hmm, I assumed we only ever wanted a single subview with each system. Why not fold the system entities into the existing subview?

Reply:
Dunno if that's the case. Done this way we get reuse of existing subviews which sounds beneficial?

Reply:
Yeah could be - @jacquese what's your opinion?

Reply:
can use however many subviews if it makes sense to separate them

Reply:
no hard and fast rules around it imo

Reply:
covered in the data flow section of the documentation https://improbableio.atlassian.net/wiki/spaces/GBU/pages/1842086036/Subviews#Data-Flow